core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/DnsResolverType.java

```diff
@@ -0,0 +1,7 @@
+package com.sequenceiq.cloudbreak.domain.stack;
+
+public enum DnsResolverType {
+    UNKNOWN,
+    LOCAL_UNBOUND,
+    FREEIPA
```

Comment:
I think it's a bit misleading, as right now FreeIPA is used only for domain level queries, and unbound is still used to handle other request and send those to the networks nameservers. I would rename this to sth like FREEIPA_FOR_ENV or whatever you see fit, as I expect that when we move forward and remove unbound alltogether there would be another value introduced here

Reply:
yes, makes sense